src/android/CameraPreviewFragment.java

```diff
@@ -232,6 +264,61 @@ public void hasFlash(HasFlashCallback hasFlashCallback) {
         hasFlashCallback.onResult(camera.getCameraInfo().hasFlashUnit());
     }
 
+    public void captureVideo(boolean useFlash, VideoCallback videoCallback) {
+       if (torchActivated) {
+           useFlash = true;
+       } else {
+           camera.getCameraControl().enableTorch(useFlash);
+       }
```

Comment:
```suggestion
         if (torchActivated) {
             useFlash = true;
         } else {
             camera.getCameraControl().enableTorch(useFlash);
         }
```

Some lines not indented properly xD